.travis.yml

```diff
@@ -10,48 +10,109 @@ env:
+       - if [ -n "${TRAVIS_TAG}" ]; then docker tag quay.io/razee/remoteresources3:${TRAVIS_COMMIT}-amd64 quay.io/razee/remoteresources3:${TRAVIS_TAG}-amd64; fi
+       - if [[ $TRAVIS_TAG =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then npm version --no-git-tag-version "${TRAVIS_TAG}"; fi
+       - docker images
+       - ./build/process-template.sh kubernetes/RemoteResourceS3/resource.yaml >/tmp/resource.yaml
+ 
+     before_deploy:
+       - docker login -u="${QUAY_ID}" -p="${QUAY_TOKEN}" quay.io
+       
+     deploy:
+       # Deploy alpha builds
+       - provider: script
+         script: docker push "quay.io/razee/remoteresources3:${TRAVIS_TAG}-amd64"
+         skip_cleanup: true
+         on:
+           tags: true
+           condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+_[0-9]{3}$
+       # Deploy released builds
+       - provider: script
+         script: docker push "quay.io/razee/remoteresources3:${TRAVIS_TAG}-amd64"
+         skip_cleanup: true
+         on:
+           tags: true
+           condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+$
+           
+   - stage: building ppc64le image
+     arch: ppc64le
+     script:
+       # Audit npm packages. Fail build whan a PR audit fails, otherwise report the vulnerability and proceed.
+       - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then npm audit; else npm audit || true; fi
+       - npm test
+       - docker build --rm -t "quay.io/razee/remoteresources3:${TRAVIS_COMMIT}-ppc64le" .
+       - if [ -n "${TRAVIS_TAG}" ]; then docker tag quay.io/razee/remoteresources3:${TRAVIS_COMMIT}-ppc64le quay.io/razee/remoteresources3:${TRAVIS_TAG}-ppc64le; fi
+       - if [[ $TRAVIS_TAG =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then npm version --no-git-tag-version "${TRAVIS_TAG}"; fi
+       - docker images
+       - ./build/process-template.sh kubernetes/RemoteResourceS3/resource.yaml >/tmp/resource.yaml
+       
+     before_deploy:
+       - docker login -u="${QUAY_ID}" -p="${QUAY_TOKEN}" quay.io
+       
+     deploy:
+       # Deploy alpha builds
+       - provider: script
+         script: docker push "quay.io/razee/remoteresources3:${TRAVIS_TAG}-ppc64le"
+         skip_cleanup: true
+         on:
+           tags: true
+           condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+_[0-9]{3}$
 
-before_deploy:
-  - docker login -u="${QUAY_ID}" -p="${QUAY_TOKEN}" quay.io
+       # Deploy released builds
+       - provider: script
+         script: docker push "quay.io/razee/remoteresources3:${TRAVIS_TAG}-ppc64le"
+         skip_cleanup: true
+         on:
+           tags: true
+           condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+$
+           
+   - stage: build and push multi-arch image
+     arch: amd64
+     script:
+       - export DOCKER_CLI_EXPERIMENTAL=enabled    
+       - if [ -n "${TRAVIS_TAG}" ]; then docker manifest create quay.io/razee/remoteresources3:${TRAVIS_TAG} quay.io/razee/remoteresources3:${TRAVIS_TAG}-amd64 quay.io/razee/remoteresources3:${TRAVIS_TAG}-ppc64le; fi
```

Comment:
```suggestion
       - if [[ $TRAVIS_TAG =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then npm version --no-git-tag-version "${TRAVIS_TAG}"; fi
       - ./build/process-template.sh kubernetes/RemoteResourceS3/resource.yaml >/tmp/resource.yaml
       - export DOCKER_CLI_EXPERIMENTAL=enabled    
       - if [ -n "${TRAVIS_TAG}" ]; then docker manifest create quay.io/razee/remoteresources3:${TRAVIS_TAG} quay.io/razee/remoteresources3:${TRAVIS_TAG}-amd64 quay.io/razee/remoteresources3:${TRAVIS_TAG}-ppc64le; fi
```

Reply:
i dont believe the file `/tmp/resource.yaml` that is created in the first (line 26) and second (line 57) stages, is accessible here in the 3rd stage.

also line 24 in the 1st stage modifies the package.json with the new version to be published. i dont believe that change will carry over to this 3rd stage.